scripts/get_access_token.py

```diff
@@ -0,0 +1,48 @@
+##!/usr/bin/env python
+
+import argparse
+import os
+from oauth2client.client import GoogleCredentials
+from oauth2client.service_account import ServiceAccountCredentials
+
+
+def get_access_token(json_credentials, new_scopes=None):
+    """I take a path to a service account's json credentials file and return an access token."""
```

Comment:
minor thing: maybe remove "I" from the comment

Reply:
haha yeah fair, I was playing around with writing from the point of view of the function but it's probably just confusing